README.md

```diff
@@ -4,6 +4,52 @@
 
 # Cellulose
 
-Cellulose is a react layout library based on contextually aware components using flexbox.
+Cellulose is a React layout library based on contextually aware components using flexbox.
 
-[//]: # (`npm i -S cellulose` or `yarn add cellulose`)
+`npm i -S cellulose` or `yarn add cellulose`
+
+## Usage
+
+Cellulose offers a responsive-style grid system
+
+```
+const breakPoints = {
+  768:  'point768',
+  1024: 'point1024'
+}
+
+return (
+  <Cellulose columns={12} breakPoints={breakPoints}>
+    <Cell spanOptions={{ 768: 6, 1024: 8 }}>
+      <div>One</div>
+    </Cell>
+    <Cell spanOptions={{ 768: 6, 1024: 4 }}>
+      <div>Two</div>
+    </Cell>
+  </Cellulose>
+)
+```
+
+With Cellulose, changing styles responsively based on breakpoints is simple
```

Comment:
It's a good idea to avoid terms like "simple", "easy", or "basic" so that people don't feel stupid if they don't find it simple.

So maybe something like, "The spanOptions can provide a number of columns for each breakpoint, or they can provide an object containing the number of columns as well as any other breakpoint-specific properties such as `className`."